### PR TITLE
Style

### PR DESCRIPTION
**Pull request by CarRamos**

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -3,8 +3,12 @@ import React, { Component } from 'react'
...
   render () {
     return (
-      <div>Footer</div>
-      
+     <footer>
+      <div>
+        <div class="header"><img src="https://www.clipartmax.com/png/full/146-1467402_clipart-info-peace-love-weed.png" height="100" width="auto" />      
+        </div>
+      </div>
+      </footer>
```
Comment (WalkerTexas-coder) [inline src/components/Footer.js:11]: Hey it looks like this isn't indented correctly please fix that up and push up your changes and I'll approve.

**Review by WalkerTexas-coder — COMMENTED** [review]
left a comment otherwise looks good.

File: src/components/Footer.js (hunk 1 above)
Comment (CarRamos) [reply src/components/Footer.js]: took a look, fixed. 100%

**Review by WalkerTexas-coder — APPROVED** [review]
Thanks you are good to go!

**Pull request merged**